index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="contact.css">
+    <title>ContactUs</title>
+</head>
+
+<body>
+    <section>
+        <div class="container">
+            <div class="contactinfo">
+                <div>
+                    <h2>Contact Us</h2>
+                    <ul class="info">
+                        <li>
+                            <span><img src="images/location.png" alt=""></span>
+                            <span>21/841/12,<br> Dr. Babaso Khanjire Society,<br>
+                                near Swami apartment<br>
+                                Jawahar Nagar(416 117)</span>
+                        </li>
+                        <li><span><img src="images/email.png" alt=""></span><span>tahir.kad*****ar@gmail.com</span></li>
+                        <li><span><img src="images/call.png" alt="">
+                            </span><span>+91 75*****167</span></li>
+                    </ul>
+
+                </div>
+                <ul class="sci">
+                    <li><a href="#"><img src="images/1.png" alt=""></a></li>
+                    <li><a href="#"><img src="images/2.png" alt=""></a></li>
+                    <li><a href="#"><img src="images/3.png" alt=""></a></li>
+                    <li><a href="#"><img src="images/4.png" alt=""></a></li>
```

Comment:
Incase web doesn't render the image then how the user will get idea about the image? Mention alt value it will help user to understand the image incase it doesn't get render.

Reply:
ok, let me add values for ALT attribute

Reply:
yes, done with it